commands/flags_parser.go

```diff
@@ -24,6 +24,10 @@ func NewCommandFlagsParser(flag *flag.FlagSet, parser FlagsParser, validator Fla
 
 // Parse parsing the args
 func (p *CommandFlagsParser) Parse(args []string) error {
+	if unknownFlags := collectUnknownFlags(p.flag, args); len(unknownFlags) > 0 {
+		return fmt.Errorf("Unknown or wrong flags: %s", strings.Join(unknownFlags, ", "))
+	}
```

Comment:
Is this check going to be executed on each parse? Why not execute it only when the parsing fails?